libbeat/outputs/discard/docs/discard.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[discard-output]]
+=== Configure the Discard output
+
+++++
+<titleabbrev>Discard</titleabbrev>
+++++
+
+The Discard output throws away data.
+
+WARNING: The Discard output should be used only for development or
+debugging issues.  Data is lost.
```

Comment:
[Doc lint]
```suggestion
WARNING: The Discard output should be used only for development or
debugging issues. Data is lost.
```

My brain runs a whitespace lint that's pretty hard to disable :P

Reply:
I learned to type on an [IBM Selectric](https://en.wikipedia.org/wiki/IBM_Selectric).  I may never be able to get over 2 spaces between sentences.

Fixed.